### PR TITLE
docs(fix): added cube cloud info on getting started docs

### PR DESCRIPTION
**Pull request by adnanrahic**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**

https://app.shortcut.com/cube-dev/story/1189/merge-getting-started-with-cube-cloud-information

### CODE REVIEW

File: docs/content/Cube-Cloud/Getting-Started/Create-new.md [hunk 1]
```diff
@@ -1,59 +1,127 @@
...
-  src="https://cube.dev/downloads/images/cube-cloud-quickstart-3.png"
+  src="https://cubedev-blog-images.s3.us-east-2.amazonaws.com/81442713-0261-424c-bb09-17b1601c10e0.png"
+  style="border: none"
+  width="100%"
+  />
+</div>
+
+### Why don't you try out some sample data?
+
+We have a sample database where you can try out Cube Cloud.
```
Comment (rpaik) [inline docs/content/Cube-Cloud/Getting-Started/Create-new.md:60]:
```suggestion
We also have a sample database where you can try out Cube Cloud.
```

**Comment by rpaik** [comment]
A few minor suggestions.

**Comment by adnanrahic** [comment]
Thanks @rpaik ! Those were great suggestions. I just applied them to the text.

**Review by hassankhan — CHANGES_REQUESTED** [review]

**Comment by rpaik** [comment]
Sorry for the delay @adnanrahic, looks good to me 😄

**Review by rpaik — APPROVED** [review]

**Pull request merged**